src/07-projects/milestones.apib

```diff
@@ -85,6 +85,7 @@ Get details for a single milestone.
                 + id: `fffaf951-b65a-4cb1-bfb6-4b93e0e60c10` (string)
             + due_on: `2018-01-01` (string)
             + name: `Initial setup` (string)
+            + description (string)
```

Comment:
Q: if the description is optional when creating a milestone, does it get a value implicitly? Otherwise we should make this property nullable.

Reply:
It's empty by default

Reply:
Is there a different between an empty string (which technically is a value) and no value (`null`) for a milestone's description in the domain? If there is no difference, I'd treat the empty string as `null`.

Reply:
In domain, there is no `null`able description. Neither in DB.
So, there is a difference I guess?

Reply:
We do not support nulls in this field. I prefer this to be transparent throughout the application and the API.

Reply:
I do think a non nullable column in the database in this case is rather a design error which should not be reflected in the domain or public API.

Let's settle on the current proposal but take this into account in the future. 😃